### ISSUE TITLE
Relocate all Web Jar dependencies

### ISSUE THREAD

**Issue by Dmitry-Matveev** [Microsoft/ApplicationInsights-Java#365]
As per [this comment](https://github.com/Microsoft/ApplicationInsights-Java/commit/799a839b59a41f56f2247a4f68c3c1cb86e18ad9#commitcomment-21670121) in the source code, the current shadow Jar process does not relocate `javax.annotation` nor `org.objectweb` causing Security Exception like the following:
```
java.lang.SecurityException: class "org.objectweb.asm.MethodAdapter"'s signer information does not match signer information of other classes in the same package
at java.lang.ClassLoader.checkCerts(ClassLoader.java:898)
at java.lang.ClassLoader.preDefineClass(ClassLoader.java:668)
at java.lang.ClassLoader.defineClass(ClassLoader.java:761)
at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
```

**Comment by debugthings** [comment]
#367 Related

**Comment by Dmitry-Matveev** [comment]
Partially addressed in 1.0.8, to review in 1.0.9.

**Comment by markpjohnson** [comment]
This is a high priority for my team. Is there any chance this will get resolved soon? We are awaiting several new features, but this issue is a blocker

**Comment by dhaval24** [comment]
@markpjohnson thank you for notifying us your blocking issue. As this is a security issue and of High P. We will discuss this in our triage meeting. It would be really helpful if you can provide us with all the other features that you are looking for in your monitoring needs so that we can put them in our list and have appropriate milestones for them. If you don't mind letting us know what is the current monitoring solution you are using or is this more of experimental phase for your team?

**Comment by dhaval24** [comment]
@Dmitry-Matveev if you have any idea that SDK is supposed to repackage every dependency that we use into the microsoft namespace. I believe we should do it to avoid any conflict with user jars in the project.

**Comment by Dmitry-Matveev** [comment]
That was the intent all along and Core Jar was simply selected as the first one to implement shadow jar. We can now think of extending this solution to the other jars we ship if there are no issues uncovered while releasing shadow Core Jar for the last couple times.

**Comment by dhaval24** [comment]
@Dmitry-Matveev as I look closely into the code this is about relocating some dependencies in the code/build.gradle which are already addressed in the code. The PR for bytecode fixes also relocates the jars for web and therefore we should be good to close this issue. Let me know.

**Comment by Dmitry-Matveev** [comment]
We can close this issue whenever PR that relocates the rest of Web dependencies is merged.

**Comment by dhaval24** [comment]
Fixed in #449
